src/config/SSSDConfig/__init__.py.in

```diff
@@ -188,6 +188,7 @@ option_strings = {
     'subdomain_enumerate' : _('Control enumeration of trusted domains'),
     'subdomain_refresh_interval' : _('How often should subdomains list be refreshed'),
     'subdomain_inherit' : _('List of options that should be inherited into a subdomain'),
+    'subdomain_homedir' : _('Define default subdomain homedir value'),
```

Comment:
Justin, feel free to ignore the suggestion ...

But what do you thiink about just "Default subdomain homedir value"?